growth/too/tasks/ztf_client.py

```diff
@@ -89,3 +92,75 @@ def ztf_references():
                          reference_filter_ids=rows['fid'].tolist(),
                          reference_filter_mags=rows['maglimit'].tolist()))
     models.db.session.commit()
+
+
+@celery.task(base=PeriodicTask, shared=False, run_every=3600)
+def ztf_depot(start_time=None, end_time=None):
+    """ZTF depot reader.
+
+    Ingests information about images from all program ids
+    (including program_id = 1) based on the nightly summary.
+    This supplements what is available from the TAP interface,
+    where information about public images is not available.
+
+    Parameters
+    ----------
+    start_time : astropy.Time
+        Start time of request.
+    end_time : astropy.Time)
+        End time of request.
+
+    """
+
+    if start_time is None:
+        start_time = time.Time.now() - time.TimeDelta(1.0*u.day)
+    if end_time is None:
+        end_time = time.Time.now()
+
+    depotdir = 'https://ztfweb.ipac.caltech.edu/ztf/depot'
+
+    mjds = np.arange(np.floor(start_time.mjd), np.ceil(end_time.mjd))
+    for mjd in mjds:
+        this_time = time.Time(mjd, format='mjd')
+        dstr = this_time.iso.split(" ")[0].replace("-", "")
+
+        url = os.path.join(depotdir, '%s/goodsubs_%s.txt' % (dstr, dstr))
+        deptable = get_ztf_depot_table(url)
+        if len(deptable) == 0:
+            continue
+
+        obs_grouped_by_jd = deptable.group_by('jd').groups
```

Comment:
Looks like this sqlalchemy table is unfortunately missing the 'jd' column, so this query cannot be completed in the same way as for ztf_obs.  Getting a ValueError when I try running it.

Reply:
Did you try a recent one? Frank updated the tables a few weeks ago maybe to make us happier ;)

Reply:
Ah, I wasn't aware, thanks for the pointer :)

Reply:
Ugh, no the 'jd' column is missing, even for the most recent S190930t. Did he update more recently than that?